draw/src/shader/draw_text.rs

```diff
@@ -661,9 +661,21 @@ impl DrawText {
                         let mut iter = glyph_infos.iter().peekable();
                         while let Some(glyph_info) = iter.next() {
                             let glyph_start = start + glyph_info.cluster;
-                            let glyph_end = start + iter.peek().map_or(string.len(), |glyph_info| glyph_info.cluster);
+
+                            let glyph_end = if let Some(next_info) = iter.peek() {
```

Comment:
I'm confused about this change. Doesn't this do exactly the same as the original line?